src/components/TimeSeriesChart.tsx

```diff
@@ -59,10 +59,14 @@ const styles = (theme: Theme) => createStyles(
             fontWeight: 'bold',
             paddingBottom: theme.spacing.unit,
         },
+        line: {
+            color: theme.palette.primary.light,
```

Comment:
Is this used?

Reply:
Let's make line colors dependent on theme. Dark theme require lighter colors, while light theme requires darker colors.